tutorials/QML_with_Classiq/QML_with_Classiq.ipynb

```diff
@@ -0,0 +1,1203 @@
+{
```

Comment:
<p>"Details" can be renamed to "Fun Exercise" or "Exercise"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Classiq/classiq-library/pull/180/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Reply:
<p>Definitely, I called it "Exercise - Two Qubits VQE", unsure why it's not visible here.</p>

Reply:
<p>I still can't view it as Exercise but maybe it's just a glitch. All good if it's visible to you. Resolving</p>